jwst/datamodels/schemas/miri_ramp.schema.yaml

```diff
@@ -1,5 +1,6 @@
 allOf:
 - $ref: core.schema.yaml
+- $ref: wcsinfo.schema.yaml
 - $ref: bunit.schema.yaml
 - type: object
   properties:
```

Comment:
I think it also needs to be added multiexposure.schema.yaml and multiproduct.schema.yaml, because they inherit core and use a SCI array, just like multislit.schema.yaml.